samza-core/src/main/scala/org/apache/samza/serializers/SerdeManager.scala

```diff
@@ -148,7 +148,10 @@ class SerdeManager(
         envelope.getSystemStreamPartition,
         envelope.getOffset,
         key,
-        message)
+        message,
+        0,
```

Comment:
Is this the size field? If so, shouldn't hardcode to 0, should use getSize instead.

Reply:
that would be a change in behavior: original code was using the instructor that hardcodes size=0

Reply:
I think that might have been an oversight. Wouldn't hurt to carry the value over.